src/site/content/en/progressive-web-apps/promote-install/index.md

```diff
@@ -0,0 +1,382 @@
+who aren't interested in installation.
+
+<div class="w-clearfix">&nbsp;</div>
+
+### Fixed header {: #header }
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="header.png" alt="Custom install button in header">
+  <figcaption class="w-figcaption">
+    Custom install button in header
+  </figcaption>
+</figure>
+
+This is an install button that is part of the header of your site. Other
+header content often includes site branding such as a logo and the hamburger
+menu. Headers may be `position:fixed` or not depending on your site's
+functionality and user needs.
+
+When used appropriately, promoting PWA install from the header of your
+site is a great way to make it easier for your most loyal customers to
+return to your experience. Pixels in your PWA header are extremely valuable,
+so make sure your install call to action is appropriately sized, of greater
+importance than other possible header content, and unintrusive.
+
+<div class="w-clearfix">&nbsp;</div>
+
+Make sure you:
+
+* Evaluate the value of your installed use case for your users. Consider
+  selective targeting to only present this promotion for users that are
+  likely to benefit from it.
+* Use precious header space efficiently. Consider what else would be helpful
+  to offer your user in the header, and weigh the priority of the install
+  promotion relative to other options.
+
+### Navigation menu {: #nav }
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="nav.png" alt="Custom install button in nav menu">
+  <figcaption class="w-figcaption">
+    Add an install button/promotion in a slide out navigation menu.
+  </figcaption>
+</figure>
+
+The navigation menu is a great place for promoting the installation of your
+app since users who open the menu are signaling engagement with your experience.
+
+Make sure you:
+
+* Avoid disrupting important navigational content. Put the PWA install
+  promotion below other menu items.
+* Offer a short, relevant pitch for why the user would benefit from
+  installing your PWA.
+
+<div class="w-clearfix">&nbsp;</div>
+
+### Landing page {: #landing }
+
+The purpose of a landing page is to promote your products & services,
+so this is one place where it can be appropriate to go large with promoting
+the benefits of installing your PWA.
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="landing.png" alt="Custom install prompt on landing page">
+  <figcaption class="w-figcaption">
+    Custom install prompt on landing page
+  </figcaption>
+</figure>
+
+First, explain your site's value proposition, then let visitors know what
+they'll get from installation.
+
+Make sure you:
+
+* Appeal to features that will matter most to your visitors and emphasize
+  keywords that might have brought them to your landing page.
+* This is a landing page. After you've made your value proposition clear,
+  make your install promotion and call to action eye catching!
```

Comment:
Shrunk and White II-19: Express coordinate ideas in similar form. That means making sure bullets start with the same grammatical structure. 

"Make your install promotion and call to action eye catching, but only after you've made your value proposition clear. This is your landing page, after all."